### PR TITLE
Add second product detail page

### PR DESCRIPTION
**Pull request by doanyennhi**
- Created a second product detail page, link recommended products to first product detail, fix description to shop name in recommended product cards, changed main product info
- Fixed bad links, renamed images of product detail 2
- Cropped images to 1:1 ratio
- Fixed date format for consistency with other product detail page

### CODE REVIEW

File: public/store/store-template/product-detail-2/index.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+            </div>
+        </div>
+        <div id="footer-copyright">
+            <p>&copy;2021 YABE</p>
+        </div>
+    </footer>
+
+    <script src="../../../js/common.js"></script>
+</body>
+</html>
```
Comment (miketvo) [inline public/store/store-template/product-detail-2/index.html:281]: Needs newline at end of file

File: public/store/store-template/product-detail-2/index.html [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+                        <a class="product-card-title" href="../product-detail">Product Title Goes Here</a>
+                        <a class="product-card-shop" href="../">Shop Name Goes Here</a>
+                        <p class="product-card-price">$12.99</p>
+                        <div class="product-card-sale-card">1/4/2017</div>
+                    </div>
+                </div>
+
+                <div class="product-card">
+                    <a href="../product-detail"><img alt="image of a product"
+                        src="../../../media/image/book_cover/heartstopper_cover.jfif"></a>
```
Comment (miketvo) [inline public/store/store-template/product-detail-2/index.html:207]: Wrong file extention

**Review by miketvo — CHANGES_REQUESTED** [review]
Please address these issues

**Review by miketvo — APPROVED** [review]
Looks good to me

**Review by rider3458 — APPROVED** [review]

**Pull request merged**